src/quo2/components/title/title.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo2.components.title.title
```

Comment:
We have a best practice for the quo2 namespaces.

Each component is in its Figma Category and its Figma title.
In this case it should be:

quo2.components.text-combinations.title.view
and the style file is then in 
quo2.components.text-combinations.title.style